crates/ibc/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 
 #![no_std]
 #![allow(clippy::large_enum_variant)]
+#![allow(clippy::result_large_err)]
```

Comment:
We should remove this [for performance reasons](https://rust-lang.github.io/rust-clippy/master/index.html#result_large_err); let's make our Error enums smaller instead.

Reply:
done